src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -191,6 +207,8 @@ public new static RevitDynamoModel Start(IRevitStartConfiguration configuration)
         private RevitDynamoModel(IRevitStartConfiguration configuration) :
             base(configuration)
         {
+            DisposeLogic.IsShuttingDown = false;
```

Comment:
If IsClosingHomeworkspace is implemented the same way as IsShuttingDown, shall we initialize it here as well?